src/AElf.Core/AElfDefaultConventionalRegistrar.cs

```diff
@@ -7,25 +8,27 @@ namespace AElf
 {
     public class AElfDefaultConventionalRegistrar : DefaultConventionalRegistrar
     {
+        private const string _interfacePrefix = "I";
+
+        private readonly List<string> _transientTypeSuffixes = 
+            new List<string>{"Service", "Provider", "Manager", "Store", "Factory"};
+
         protected override ServiceLifetime? GetServiceLifetimeFromClassHierarcy(Type type)
         {
             var lifeTime = base.GetServiceLifetimeFromClassHierarcy(type);
-            if (lifeTime != null)
+            if (lifeTime == null)
```

Comment:
in your implements, `var lifeTime = base.GetServiceLifetimeFromClassHierarcy(type);` did nothing, remove it will get the same result

if you do not know the details of how these work, please do not change it.

What's the meanning of `private const string _interfacePrefix = "I";`? Will you use other instead of using "I"?